tests/phpunit/Util/MockupFactory.php

```diff
@@ -282,9 +282,25 @@ protected function getSkinStub() {
 		$skin->expects( $this->testCase->any() )
 			->method( 'msg' )
 			->will( $this->testCase->returnValue( $this->getMessageStub() ) );
+
+		$skin->expects( $this->testCase->any() )
+			->method( 'getConfig' )
+			->will( $this->testCase->returnValue( $this->getConfigStub() ) );
+
 		return $skin;
 	}
 
+	/**
+	 * @return \PHPUnit\Framework\MockObject\MockObject|\PHPUnit_Framework_MockObject_MockObject
+	 */
+	protected function getConfigStub() {
+		$config = $this->testCase->getMockBuilder( '\Config' )
+			->disableOriginalConstructor()
+			->getMock();
+
+		return $config;
```

Comment:
return $this->testCase->createMock( \Config::class );

Reply:
That doesn't work in the mock factory. I tried it last time and also got this:
![image](https://user-images.githubusercontent.com/1428594/120116183-29be1180-c187-11eb-9116-c0b756242604.png)

Reply:
Right ok